### PR TITLE
Chi 305 improve responsive behavior of step component

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/196/ for more details.

File: src/website/views/components/steps.pug [hunk 1]
```diff
@@ -258,3 +258,127 @@ p.-text
       </li>
     </ul>
 
+h4 Responsiveness
+p.-text
+  | You can use a modifier class with format <code>-{breakpoint}--labels</code> to make responsive the steps component.
```
Comment (mattnickles) [inline src/website/views/components/steps.pug]: Change text to:
Use `-labels-{breakpoint}--hide` to hide step component labels on specific breakpoints or `-labels--hide` to hide on all breakpoints. Breakpoints supported are `sm`, `md`, `lg`, and `xl`. Labels are shown on all breakpoints by default.

Comment (davidojedacl) [reply src/website/views/components/steps.pug]: done

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/204/ for more details.

**Pull request merged**